fastlane/swift/Runner.swift

```diff
@@ -147,6 +204,7 @@ extension Runner : SocketClientDelegateProtocol {
             self.socketClient = nil
             verbose(message: "connection closed!")
             if self.shouldLeaveDispatchGroupDuringDisconnect {
+                print("Leave \(#function) \(#line)")
```

Comment:
Either delete or 👇 ?

```suggestion
                verbose(message: "Leave \(#function) \(#line)")
```